db/migrate/20160729182517_remove_system_add_source_to_miq_ae_namespace.rb

```diff
@@ -0,0 +1,31 @@
+class RemoveSystemAddSourceToMiqAeNamespace < ActiveRecord::Migration[5.0]
+  class MiqAeNamespace < ActiveRecord::Base; end
+
+  def up
+    say_with_time('Migrating System attribute to Source in MiqAeNamespace') do
+      add_column :miq_ae_namespaces, :source, :string
+
+      MiqAeNamespace.where('parent_id IS NULL').each do |obj|
+        source = if obj.name == 'ManageIQ'
+                   'system'
+                 else
+                   obj.system ? 'user_locked' : 'user'
```

Comment:
Don't embed a ternary in the else of a conditional...it just complicates the if/else

```ruby
if obj.name == "ManageIQ"
  "system"
elsif obj.system
  "user_locked"
else
  "user"
end
```

is much clearer.